src/apps/Console/Console.js

```diff
@@ -11,45 +11,30 @@ import {
   useToast,
   GU,
 } from '@aragon/ui'
-import { hash as namehash } from 'eth-ens-namehash'
-import { encodeFunctionCall } from 'web3-eth-abi'
-import ConsoleFeedback from './ConsoleFeedback'
+import { performTransactionPaths } from '../../aragonjs-wrapper'
 import {
-  parseCommand,
-  parseMethodCall,
-  parseInitParams,
-  parsePermissions,
-} from './console-utils'
+  getInstallTransactionPath,
+  getExecTransactionPath,
+  getActTransactionPath,
+} from './CommandHandlers'
+import ConsoleFeedback from './ConsoleFeedback'
+import { parseCommand } from './console-utils'
 import IconPrompt from './IconPrompt'
 import KEYCODES from '../../keycodes'
 import { clamp } from '../../math-utils'
 import { AragonType, AppType } from '../../prop-types'
-import { encodeFunctionCallFromSignature } from './web3-encoding-utils'
-import { useWallet } from '../../wallet'
-// Maximum time (in milliseconds) to wait for a response
-// from the aragon.js apm repo content fetcher
-const REPO_FETCH_TIMEOUT = 3000
-const APP_POSTFIX = '.aragonpm.eth'
+
+const HISTORY_ARRAY = 'HISTORY_ARRAY'
```

Comment:
We should use a more descriptive name here, something like `CONSOLE_COMMAND_HISTORY`, and we should suffix the variable with `_KEY`.